examples/lighting-app/k32w/main/AppTask.cpp

```diff
@@ -489,7 +489,7 @@ void AppTask::BleHandler(AppEvent * aEvent)
     {
         ConnectivityMgr().SetBLEAdvertisingEnabled(true);
 
-        if (OpenBasicCommissioningWindow(chip::ResetFabrics::kNo) == CHIP_NO_ERROR)
+        if (chip::Server::GetServer().OpenBasicCommissioningWindow(chip::ResetFabrics::kNo) == CHIP_NO_ERROR)
```

Comment:
@msandstedt  I was thinking of trying that for ota-requestor-app, since `Server.cpp` doesn't support adding protocol message handlers, and I need to handle BDX messages.